src/OmniKassa/OmniKassa.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;netstandard2.0;netstandard2.1;net452;net5.0</TargetFrameworks>
-    <Version>1.3.7</Version>
+    <TargetFrameworks>netstandard1.3;netstandard2.0;netstandard2.1;net462;net5.0;net6.0</TargetFrameworks>
```

Comment:
consider removing net5.0 and the "net5.0\OmniKassa.xml" configuration section below.

Reply:
that is a good one